torch/_inductor/compile_fx.py

```diff
@@ -1173,7 +1173,7 @@ def log_graph_runnable() -> str:
                                 "pt2_configs": str(get_patched_config_dict())
                             }
                         )
-                    except ValueError:
+                    except:
                         # TODO(T216453900): need to work around for now to support vllm
                         # See details in vllm/compilation/pass_manager.py.
                         log.warning("failed to log pt2_configs")
```

Comment:
We may want to include more info here, pass exc_info=True to determine what error type was raised?